src/main/java/cz/brmlab/yodaqa/flow/dashboard/AnswerSourceStructured.java

```diff
@@ -0,0 +1,14 @@
+package cz.brmlab.yodaqa.flow.dashboard;
+
+/**
+ * Created by nhl on 7/10/15.
+ */
```

Comment:
We have git for authorship tracking. Actual description of the class (or nothing if it's obvious) is preferred. :)